debian/enterprise-control-center/Dockerfile

```diff
@@ -33,7 +33,7 @@ ENV CONTROL_CENTER_CONFIG_DIR=/etc/confluent-${COMPONENT}
 EXPOSE 9021
 
 RUN echo "===> Installing ${COMPONENT}..." \
-    && apt-get update && apt-get install -y confluent-${COMPONENT}=${CONFLUENT_VERSION}-${CONFLUENT_DEB_VERSION} \
+    && apt-get update && apt-get install -y confluent-${COMPONENT}=${CONFLUENT_VERSION}${CONFLUENT_PLATFORM_LABEL}-${CONFLUENT_DEB_VERSION} \
```

Comment:
This is overridden by local.make in our private repo, and is combined with the revision to form the semver label for the target platform. Debian snapshots end up looking like "~SNAPSHOT-1", while RPMs look like ".0.1.SNAPSHOT". For non-snapshot cases it's just empty.